### PR TITLE
Add test for Authorization for TCP traffic

### LINKED ISSUE #6578: [istio.io testing] Need test for Authorization for TCP Traffic
**Describe the feature request**
We need to develop a test for the [Authorization for TCP traffic](https://preliminary.istio.io/docs/tasks/security/authorization/authz-tcp/) task in order to provide automated testing of the Istio.io website for future releases.

Example tests and the documentation for the framework can be found here: https://github.com/istio/istio/tree/master/tests/integration/istioio

A YouTube video describing the usage of the framework can be found here: https://www.youtube.com/watch?v=3y-z8NaVwr0

Please reach out to the Test and Release channel on the [Istio Slack](https://discuss.istio.io/t/istio-slack-channel/1527) or [Test and Release Discuss](https://discuss.istio.io/c/test-and-release)  with any questions.

[ ] Configuration Infrastructure
[x ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[x ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Additional context**

### PR DESCRIPTION
**Pull request by shamsher31**
To fix #6578

### CODE REVIEW

**Comment by fpesce** [comment]
@shamsher31 would you mind rebasing your PR? Thanks!

File: tests/common/scripts/bookinfo.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Step 1: setup bookinfo & sleep pods
```
Comment (elysdir) [inline tests/common/scripts/bookinfo.txt]:
```suggestion
# Step 1: set up bookinfo & sleep pods
```

File: tests/security/scripts/authz_tcp_traffic.txt [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+# $snippet wait_for_product_destination_rule_1.sh syntax="bash"
+$ istioctl experimental wait --for=distribution DestinationRule productpage.default --timeout=60s
+# $endsnippet
+
+# Step 3: update the reviews workload to only use the v2 of the ratings workload
+# $snippet update-reviews-v2.sh syntax="bash" outputis="text"
+$ kubectl apply -f @samples/bookinfo/networking/virtual-service-ratings-db.yaml@
+# $endsnippet
+
+# Step 4: Verify ratings is unavailable
```
Comment (elysdir) [inline tests/security/scripts/authz_tcp_traffic.txt]: I'm not quite sure what the phrase "Verify ratings is unavailable" means--I can think of a couple of different things that it might mean. Can you rephrase slightly for clarity?

This question also also applies to step 8, below. ("Verify ratings is unavailable again")

**Comment by frankbu** [comment]
Closing, PR is obsolete. A test for this task has already been added.
